htsget-search/src/htsget/mod.rs

```diff
@@ -98,6 +98,8 @@ impl From<StorageError> for HtsGetError {
         Self::InvalidInput(format!("Wrong key derived from ID: {}", key))
       }
       StorageError::IoError(e, key) => Self::IoError(format!("Io error: {}, from ID: {}", e, key)),
+      #[cfg(feature = "aws")]
+      StorageError::AwsError { .. } => Self::IoError(format!("AWS error: {:?}", err)),
```

Comment:
AWS S3 error... we are not using other storage backends than S3 on AWS... for now? ;)